app/views/site/partials/_eez_map.html.erb

```diff
@@ -11,13 +11,13 @@
     <% end %>
     <v-map-interactive
       :id="'eez-map'"
-      :has-download-button="true"
+      :nload-button="true"
```

Comment:
nload? Is that correct?

Reply:
A failed attempt to delete the whole line! 🙃